Surveys pricing faq (#6957)
PostHog/posthog.com: contents/docs/surveys/troubleshooting.mdx

posthog.getActiveMatchingSurveys(surveys => console.log(surveys))
```

## How are surveys priced?

Surveys are priced based on the [number of responses](/pricing?product=surveys). However, because surveys is a tool that utilizes other parts of PostHog, such as events and feature flags, it's important to highlight the underlying costs too.

> Whenever a response is sent, it's stored as an event in PostHog, along with survey "shown" and "dismissed" events. Events are free up to the 1st million per month, and then $0.00031 after that. This should be negligible unless you're sending surveys to millions of users a month.

Reviewer: we generally try to limit using actual pricing figures anywhere because they are liable to change. 

```suggestion
Surveys incur additional costs in two ways:

1. Events are stored when surveys are shown or dismissed, and when a response is submitted.
2. Feature flags are used when user property targeting or linked flags are used in a survey.

We have very generous free volumes for events and feature flags every month, and beyond the free tier costs per event or flag request are negligible. Check our our [pricing page](/pricing) for exact figures, but for 10,000 responses per month you can expect to pay less than $10 in other costs.

If you use feature flags outside of just surveys, you likely won't incur any additional costs, as flag API calls contain all flags needed for the session.
```

@joethreepwood can prob make that more succinct. And i'm not sure about the accuracy of the last sentence but i think that is largely true? or something along those lines.

Author: > If you use feature flags outside of just surveys, you likely won't incur any additional costs, as flag API calls contain all flags needed for the session.

It's priced based on decide requests so this would still incur additional costs if they used flags as part of feature flags right? Unless you meant something else?

Reviewer 2: No additional costs _for surveys_

Reviewer 2: that if you're already using feature flags, you wouldn't be charged extra for using surveys with feature flags. (since generally there are no extra decide requests being made)

Buuut, if you're not using feature flags, then there's an extra charge for using surveys with feature flags.

Author: errr I thought those scenarios were supposed to be flipped? Where if you have existing feature flags already, survey flags are charged because there's no way to differentiate the different types of flag requests. But if you don't use feature flags at all, then survey flags are NOT charged for

Reviewer 2: Sorry, that was very poorly worded on my side, I was thinking of the linked flag case, but in that case it means you're already using flags, so it's wrong.

You're right, if you don't use flags, survey user targeting flags aren't charged.

But if you're already using flags, you're charged _ONLY_ if you make extra requests for flags for surveys, which doesn't happen by default.

Author: Well, since we decided to go with charging for dependent flags usage at all times, we can leave the last paragraph here out totally then!

Reviewer 2: > If you use feature flags outside of surveys, you likely won't incur any additional flag costs, as flag API calls contain all flags needed for the session.

This is still valid though 😅 ; there's no extra charge if you already have flags, unless you make extra requests

Author: surveys can make extra requests though no?? Either way I think it's more confusing to have that statement than to not

Reviewer 2: they don't currently, since `getFeatureFlag()` doesn't mean an extra request, and surveys are loaded after decide response.

I think it's useful to have this in because it reassures users that there's no exorbitant _extra_ fees for using flags with surveys. I'm not sure what the confusion is here, mind elaborating?

Author: As a user I feel like this statement `10,000 survey responses per month would typically cost less than $10 per month for events and flags, for example.` is what reassures me and explains the situation best already

With the last statement, do we mean something like "flags don't get double charged in feature flags pricing if they're used in surveys" <-- something like that?? That still sounds confusing to me

Reviewer 2: Fair 👍

Author: we can always add it if people are confused!

> Whenever user properties targeting or linked flags are used in a survey, feature flags are used underneath the hood. Feature flags are free up to the 1st million requests per month, and then $0.0001 after that.

Author: I did not mention "surveys flags are free if you don't have any other flags" because there isn't a losing point for the customer in that scenario that they need to be aware of

> Whenever user properties targeting or linked flags are used in a survey, feature flags are used underneath the hood. Feature flags are free up to the 1st million requests per month, and then $0.0001 after that.

Reviewer: i'd remove this as a separate line and combine as i suggested above.

> We want to make sure that our customers use surveys without any hidden costs because surveys was built in a way that allows maximum flexibility with the rest of what PostHog offers. For example, because responses are stored as events, you can use PostHog [insights](/docs/features/insights) to analyze your survey responses even further, and to easily export data. Or if you want to create cohorts of users who've responded to a survey in a certain way, you can do that too with cohorts and feature flags.

Reviewer: ```suggestion
### Why does surveys work with events and feature flags?

Building surveys in this way allows maximum flexibility and insight by combining with the rest of what PostHog offers. For example, because responses are stored as events, you can use PostHog [insights](/docs/features/insights) to analyze your survey responses even further, and to easily export data. Or if you want to create cohorts of users who've responded to a survey in a certain way, you can do that with cohorts and feature flags.
```

Reviewer 3: ```suggestion
### Why do surveys use events and feature flags?
We built surveys in this way to enable maximum flexibility and ensure it combines well with other PostHog tools. 

Storing responses as events means you can use them within insights or cohorts, for example. Unlike other tools, PostHog doesn't limit you to analyzing responses in a single way. Want to create a feature flag that targets users who responded to a survey in a particular way, or breakdown responses by geography? PostHog does that!
```

> If you have any questions about pricing or if the underlying costs of your surveys usage supersedes your default usage for your other PostHog subscriptions, please reach out to us through any of these [support options](https://posthog.com/docs/support-options).

Reviewer: ```suggestion
If you have any questions about pricing, please reach out to us through any of these [support options](https://posthog.com/docs/support-options).
```

Reviewer 3: If you have any questions about pricing, please [let us know](/questions).

posthog.getActiveMatchingSurveys(surveys => console.log(surveys))
```

## How are surveys priced?

Surveys are priced based on the [number of responses](/pricing?product=surveys) received each month. However, because surveys utilize other parts of PostHog, such as events and feature flags, it's important to highlight the underlying costs too.

Surveys incur additional costs in two ways:

1. Surveys create events when shown, dismissed, and responded to.
2. Targeting surveys by user properties or linked flags uses a feature flag.

We have generous free allowances for events and feature flags each month, and [costs per event or flag request](/pricing) are negligible beyond this. 10,000 survey responses per month would typically cost less than $10 per month for events and flags, for example.

> If you use feature flags outside of surveys, you likely won't incur any additional flag costs, as flag API calls contain all flags needed for the session.

Author: ```suggestion
```
